### DISCUSSION TITLE
Strelka alert help needed

### DISCUSSION THREAD

**Discussion by ben-sec** [Security-Onion-Solutions/securityonion#7672] [Unsupported Versions] [answered]
Hello!

Today I got the Strelka alert "SUSP_Excel4Macro_AutoOpen" on my forward node for a file with the MD5 Hash b350f4c8d15d16b2a6905b8fcbc2bb62. This MD5 is, according to 31 vendors on Virus Total, highly malicious. The file.source of the Strelka alert says "/dev/shm/tmpm2ow384r". But what host is it referring to? The agent.name (in this case my forward node)? I have the directory   /dev/shm on this node, but it's empty. One more hint: The request.client of the alert says "go-filestream". Between what IPs?

Your help is highly appreciated to hunt this one down. Thanks in advance!

Cheers, Ben

**Comment by defensivedepth** [comment, marked as answer]
Under that alert, you should see a field called `request.attributes.filename`

That will have something like `/nsm/strelka/processed/HTTP-FeTYiurOBvCULOJ49-347ghegh.exe`


In that alert, you will also see a field called `log.id.fuid` - if you right-click and go to Actions --> Correlate, that should bring up all related logs

**Reply by ben-sec** [reply]
Thanks, that helped a lot!
